### ISSUE TITLE
windows: distributed query 

### ISSUE THREAD

**Issue by saylid** [facebook/osquery#4261]
distributed query, "select * from programs" is not returning any result, where as if I run same query using osqueryi it returns results in very few seconds.

**Comment by muffins** [comment]
@saylid this sounds like it might be more of an environment setup issue. Can you give us more information regarding what backend you're using?

**Comment by muffins** [comment]
@saylid you might wanna ask the folks in #kolide on Slack, or even try cutting an issue on the [Kolide github](https://github.com/kolide/fleet/issues). Can you paste your sample configuration and your distribtued query that you're attempting to run?

**Comment by groob** [comment]
This is the issue in fleet I believe https://github.com/kolide/fleet/issues/1735

If there's an issue with distributed query, we need to see logs. I'm happy to help troubleshoot either in the fleet issue above or in osquery slack before deciding there's a issue with osquery itself. @muffins feel free to close this issue so we can troubleshoot upstream.

**Comment by Jackbennett** [comment]
@saylid put the semicolon at the end of the distributed query, if that works add to [my fleet issue](https://github.com/kolide/fleet/issues/1712)

**Comment by saylid** [comment]
@Jackbennett tried by putting semicolon, that didn't work.

**Comment by saylid** [comment]
@groob yes i am facing the same issue you mentioned. It is the same like After querying 'services' it doesn't call to distributed/write, just continue with distributed/read. After this, any table I query will not call distributed/write again. 
@muffins As this is closed where will i get the updates about this issue?

**Comment by muffins** [comment]
@saylid we're already tracking an issue with distributed queries against the `services` table in #4187, so let's track status updates there.
